astro/src/content/docs/lifecycle/authenticate-users/passwordless/magic-links.mdx

```diff
@@ -276,21 +289,19 @@ Consult the [tenant API documentation](/docs/apis/tenants) for the length limits
 
 ## Security
 
-With magic link authentication, if the user's email account is hijacked, their account on your system is compromised. However, many organizations have security policies and protections around email accounts. It is often easier to protect and regularly change one email account password than to change all of a user's passwords. Email accounts are also more likely to have two-factor authentication enabled.
+With magic link authentication, if the user's email account or phone number is hijacked, their account on your system is compromised. However, many organizations have security policies and protections around email accounts. It is often easier to protect and regularly change one email account password than to change all of a user's passwords. Email accounts are also more likely to have two-factor authentication enabled.
 
-One way to increase the security of your magic link authentications is to decrease the lifetime of the code. This will help if the email is compromised or accidentally forwarded.
+One way to increase the security of your magic link authentications is to decrease the lifetime of the code. This will help if the email or SMS message is compromised or accidentally forwarded.
 
 There are no limits on how many passwordless requests can be made for a user, but only the most recent code is valid. Using any of the others, even if they have not yet expired, will display an `Invalid login credentials` message to the user.
 
-If someone tries to log in with an email that is not present in the FusionAuth user database, they'll see the same notification as they would if the email existed. No email will be sent.
+If someone tries to log in with an email or phone number that is not present in the FusionAuth user database, they'll see the same notification as they would if the email existed (NOT TRUE FOR PHONE). No email will be sent.
 
 If you use the passwordless API, follow the principle of least privilege, and limit calls to which the API key has access. If you are using the API key only for magic link passwordless login, don't give this key any other permissions.
 
 ### What About Users' Passwords
 
-When FusionAuth is your user datastore, adding a user means you must either provide a password or send them a link to set up their initial password.
-
-If you are only allowing magic link authentication for your application, don't allow the user to specify a password and instead generate a random series of characters for the password. We recommend generating at least 32 characters in the ASCII character set that are completely random to ensure the user's account is secure.
+When FusionAuth is your user datastore, you can optionally choose to disable passwords at the tenant level to make the passwordless login process easier. See the [Password enabled](/docs/get-started/core-concepts/tenants#password) tenant setting.
```

Comment:
not directly related to this, but now that we don't require passwords of users, they no longer need to work around this.